### ISSUE TITLE
Windows 10 UWP Cordova App

### ISSUE THREAD

**Issue by napcat** [Microsoft/cordova-plugin-code-push#231]
Hello,

Will it be possible to have support for UWP Apps any time soon?

We have a lot of apps published for all the current mobile platforms,  and it's getting crazy to keep up with all the stores updates and validations times. It would be a killer feature if this plugin could support UWP apps.

Thank you.

**Comment by davidofwatkins** [comment]
Has there been any further consideration for a Cordova CodePush client for UWP apps? This would be very useful for WIndows apps! :)

**Comment by digaus** [comment]
Agreed, plugin from Microsoft without support for Windows? Really would be awesome because I taregt Windows 10 (UWP) on Tablets etc.

**Comment by iageoghe** [comment]
This is something that we are looking at, but currently don't have a timeline for completing. When the plan comes together we will update you here, on the Mobile Center [roadmap](https://docs.microsoft.com/en-us/mobile-center/general/roadmap), or on our [blog](https://microsoft.github.io/code-push/blog/index.html).

**Comment by joels94** [comment]
Looking forward for better support for Cordova based UWP apps in App Center.
